subgraph.yaml

```diff
@@ -32,11 +32,11 @@ dataSources:
       file: ./src/mapping.ts
   - kind: ethereum/contract
     name: Badges
-    network: goerli
```

Comment:
that means no goerli deploys anymore?

Reply:
This is an auto-updated file that gets updated based on what network we're trying to build it for. `yarn build:goerli` will update the entries here with `goerli` as the network name in this file